views/book_detail.pug

```diff
@@ -3,11 +3,11 @@ extends layout
 block content
   h1 Title: #{book.title}
 
-  p #[strong Author:]
+  p #[strong Author: ]
```

Comment:
Did you check the rendering of this? I.e. that space might have been for layout.

Reply:
Yes, I’ve checked the rendering. The space is needed for the hyperlink as the only child element in the next line in the pug template. One could also insert a non-breaking space after the brackets: p #[strong Author:]&nbsp;